src/util/snippets.py

```diff
@@ -11,6 +12,11 @@
 from psycopg2.extensions import quote_ident
 from psycopg2.extras import Json
 
+try:
+    from odoo.sql_db import db_connect
+except ImportError:
+    from openerp.sql_db import db_connect
```

Comment:
You can fully ignore openerp cases, this script won't work in old code anyway.
```suggestion
    pass
```